bus/bus.go

```diff
@@ -434,7 +434,7 @@ func (b *Bus) Handler() http.Handler {
 		"GET    /contract/:id/roots":     b.contractIDRootsHandlerGET,
 		"GET    /contract/:id/size":      b.contractSizeHandlerGET,
 
-		"GET    /hosts":                          b.hostsHandlerGETDeprecated,
+		"POST   /hosts":                          b.hostsHandlerPOST,
```

Comment:
@alexfreska This PR requires a UI update. `/hosts` behaves as `/search/hosts` did now and is `POST` instead of `GET`.